rules/typescript.js

```diff
@@ -36,10 +37,17 @@ const typescriptRules = {
       selector: 'default',
       format: ['camelCase'],
     },
-
     {
       selector: 'variable',
-      format: ['camelCase', 'UPPER_CASE'],
+      format: ['camelCase', 'UPPER_CASE', 'PascalCase'],
+    },
+    {
+      selector: 'property',
+      format: ['camelCase', 'UPPER_CASE', 'PascalCase'],
+    },
+    {
+      selector: 'enumMember',
```

Comment:
Is there one for enum declaration?

Reply:
yes. that should be PascalCase, right?

Reply:
Yup! That's the convention I've been following.